modules/installation-launching-installer.adoc

```diff
@@ -312,6 +317,11 @@ endif::vsphere,vmc[]
 .. Enter a descriptive name for your cluster.
 ifdef::vsphere,vmc[]
 The cluster name must be the same one that you used in the DNS records that you configured.
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```

Comment:
See previous.

Reply:
I have incorporated the above suggested changes! :)